ggml/src/ggml-rpc.cpp

```diff
@@ -96,6 +96,17 @@ enum rpc_cmd {
     RPC_CMD_COUNT,
 };
 
+#pragma pack(1)
+struct request_alloc_buffer {
```

Comment:
I would prefer to prefix these with some common prefix. For example:

```c
struct ggml_rpc_msg_alloc_buffer_req;
struct ggml_rpc_msg_alloc_buffer_rsp;
```

Reply:
ok, I will go with the last suggestion